### ISSUE TITLE
[20260403] MOMO: Mars Orbital Model Foundation Model for Mars Orbital Applications

### ISSUE THREAD

**Issue by thinson** [thinson/RS-PaperClaw#269]
# [20260403] MOMO: Mars Orbital Model Foundation Model for Mars Orbital Applications

## 📋 基础信息

| 项目 | 内容 |
|------|------|
| **标题** | MOMO: Mars Orbital Model Foundation Model for Mars Orbital Applications |
| **作者** | Purohit Mirali, Gajera Bimal, Mehta Irish, Tokas Bhanu, Adler Jacob et al. |
| **日期** | 2026-04-03 |
| **arXiv** | [abs](https://arxiv.org/abs/2604.02719v1) \| [pdf](https://arxiv.org/pdf/2604.02719v1) |
| **TL;DR** | MOMO是首个多传感器火星遥感基础模型，通过等验证损失策略融合HiRISE、CTX和THEMIS三种传感器数据，在火星基准测试的9项下游任务中表现优于现有基线。 |
| **摘要** | 我们介绍了 MOMO，这是首个用于火星遥感的多传感器基础模型。MOMO 采用模型融合技术，整合了从三种关键火星传感器（HiRISE、CTX 和 THEMIS）独立学习到的表征，其分辨率范围从 0.25 米/像素到 100 米/像素。我们方法的核心是新颖的等验证损失（EVL）策略，该策略在通过任务算术进行融合之前，基于验证损失相似性对齐不同传感器的模型检查点。这确保了模型在兼容的收敛阶段进行融合，从而提高稳定性和泛化能力。我们在从火星轨道数据中精选的大规模高质量语料库（约 1200 万个样本）上训练 MOMO，并在 Mars-Bench 的 9 个下游任务上对其进行评估。与 ImageNet 预训练、地球观测基础模型、传感器特定预训练和全监督基线相比，MOMO 取得了更好的整体性能。特别是在分割任务上，MOMO 表现出持续且显著的性能提升。我们的结果表明，通过最优检查点选择策略进行模型融合，... |
| **标签** | 20260403, Mars Remote Sensing, Foundation Model, Multi-Sensor Fusion, Model Merge, Mars Orbital Data |

---

## 🧠 TL;DR（速览）

- MOMO是首个多传感器火星遥感基础模型，通过等验证损失策略融合HiRISE、CTX和THEMIS三种传感器数据，在火星基准测试的9项下游任务中表现优于现有基线。

---

## 📸 论文概览

<table><tr><td><img src="https://raw.githubusercontent.com/thinson/RS-PaperClaw/main/papers/previews/2604.02719v1/page_1.jpg" width="100%"/><br/>Page 1</td><td><img src="https://raw.githubusercontent.com/thinson/RS-PaperClaw/main/papers/previews/2604.02719v1/page_2.jpg" width="100%"/><br/>Page 2</td><td><img src="https://raw.githubusercontent.com/thinson/RS-PaperClaw/main/papers/previews/2604.02719v1/page_3.jpg" width="100%"/><br/>Page 3</td></tr></table>

---

### Q1: 本文主要解决什么问题？
<回答>

🎯 结论：本文主要解决如何构建首个能够高效处理火星多传感器、多分辨率遥感数据的基础模型问题。

📌 要点：
- **背景**：火星轨道传感器（HiRISE、CTX、THEMIS）具有极大的分辨率差异（0.25m/pixel至100m/pixel）和覆盖范围差异（CTX覆盖近100%火星表面，HiRISE仅覆盖<3%），数据异构性强
- **痛点**：现有地球观测基础模型的技术路线（数据堆叠或异构数据混合）无法直接迁移到火星场景——堆叠需要大量时空重合观测且传感器分辨率需相近，混合则需要在新传感器加入时重新训练整个模型
- **本文目标**：提出一种无需直接组合异构数据的多传感器基础模型构建方法，通过独立预训练+模型合并策略，实现跨分辨率、跨传感器的统一表征学习

### Q2: 前人技术路线？
<回答>

🎯 结论：前人技术路线主要包括地球观测基础模型的数据堆叠策略、异构数据混合策略，以及针对火星的单一传感器预训练方法。

📌 要点：
- **路线一：多传感器数据堆叠**（如SatMAE、Scale-MAE、Presto）：将时空对齐的多传感器数据作为不同通道堆叠到单一编码器，或为每个传感器设计独立分词器
- **路线二：异构数据混合训练**（如Satlas）：将多传感器数据混合到单一预训练数据集中进行联合训练
- **路线三：模型编辑/合并技术**（如权重平均、任务算术）：通过简单权重平均、层匹配、软对齐等方式合并多个微调模型的权重，或基于验证损失对齐的模型融合方法
- **路线四：火星单一传感器预训练**（如Purohit et al. 2023）：仅在CTX数据（100万张图像块）上进行预训练，评估限于2个下游任务

### Q3: 前人方案的局限性？
<回答>

🎯 结论：前人方案的核心局限性在于无法有效处理火星遥感数据的极端异构性，且缺乏系统性的多传感器融合策略。

📌 要点：
- **数据堆叠策略的局限**：需要大量时空重合观测，且要求传感器分辨率相近（如10-30m/pixel）；火星传感器分辨率差异达400倍（0.25m vs 100m），覆盖范围差异巨大，无法满足堆叠条件
- **异构数据混合策略的局限**：每当新传感器加入时必须重新训练整个模型，计算成本高且灵活性差；无法利用各传感器已独立训练好的模型
- **现有模型编辑方法的局限**：通常直接在最终检查点合并模型，未考虑不同数据分布下模型训练轨迹的差异，导致融合不稳定；缺乏针对多分辨率遥感数据的系统性检查点选择策略
- **火星单一传感器预训练的局限**：仅覆盖单一传感器（CTX）和单一分辨率，无法泛化到其他传感器（HiRISE、THEMIS），应用场景受限

### Q4: 核心思路？
<回答>

🎯 结论：本文提出"分而治之，择优而合"的策略——先为每个传感器独立训练模型，再通过验证损失对齐选择最优检查点，最后用任务算术合并成统一基础模型。

📌 要点：
- **独立预训练阶段**：为HiRISE（高分辨率光学）、CTX（中分辨率光学）、THEMIS（热红外）三个传感器分别训练独立的Masked Autoencoder模型，让各模型先学习各自传感器的独特分布和特征
- **检查点选择阶段**：提出Equal Validation Loss（EVL）策略，遍历各传感器的训练检查点，寻找验证损失值相近的组合，确保模型在"同等收敛程度"下进行融合
- **模型合并阶段**：使用任务算术（task arithmetic）将选定的检查点权重融合，得到最终的MOMO模型
- **损失函数优化**：设计多目标损失函数（MSE+SSIM+LPIPS+梯度损失），兼顾像素级重建与感知/结构一致性，特别适合重建火星地貌特征（撞击坑边缘、滑坡边界等）

### Q5: 方法亮点？
<回答>

🎯 结论：本文方法在检查点选择策略、损失函数设计和多传感器融合范式上均有创新。

📌 要点：
- **亮点一：Equal Validation Loss（EVL）检查点选择策略**：首次提出基于验证损失相似性的跨传感器检查点对齐方法，解决不同分辨率数据训练轨迹差异导致的融合不稳定问题，是首个系统化的任务算术检查点选择策略
- **亮点二：感知-结构联合损失函数**：针对MSE仅关注像素级强度匹配的缺陷，引入SSIM（结构相似性）、LPIPS（感知损失）和梯度损失，显著提升火星地貌结构细节（撞击坑形状、边界连续性）的重建质量
- **亮点三：模型合并范式用于多分辨率遥感**：开创性地将模型编辑/合并技术应用于构建多传感器基础模型，避免了直接处理异构数据的困难，为新传感器灵活接入提供可扩展框架
- **亮点四：首个火星遥感基础模型**：MOMO是首个系统开发、分析和评估的火星轨道应用基础模型，填补该领域空白

### Q6: 主要贡献？
<回答>

🎯 结论：本文从理论、方法和实验三个维度为火星遥感基础模型研究奠定了重要基础。

📌 要点：
- **理论贡献**：提出验证损失对齐的模型融合理论框架，证明通过选择兼容收敛阶段的检查点可以提升多分辨率数据融合的稳定性与泛化性
- **方法贡献**：
  - 设计EVL最优检查点选择算法，实现跨传感器模型的稳定融合
  - 构建针对火星地貌特征的多目标重建损失函数
  - 建立可扩展的多传感器基础模型构建流程
- **实验贡献**：
  - 在约1200万样本的大规模火星轨道数据集上完成预训练
  - 在Mars-Bench全部9个下游任务上完成系统评估
  - 证明MOMO在分割任务上显著优于ImageNet预训练、地球观测基础模型（msGFM等）、传感器特定预训练及全监督基线

### Q7: 实验数据集？
<回答>

🎯 结论：实验使用约1200万样本的火星轨道数据集进行预训练，在Mars-Bench的9个下游任务上评估，MOMO整体性能最优，分割任务提升尤为显著。

📌 要点：
- **预训练数据集**：约1200万个样本，来自HiRISE（0.25m/pixel）、CTX（~6m/pixel）、THEMIS（100m/pixel）三个关键火星传感器
- **评估基准**：Mars-Bench的全部9个轨道下游任务，涵盖分类和分割两类任务
- **对比基线**：
  - ImageNet预训练模型
  - 地球观测基础模型（EO-FMs）
  - 传感器特定预训练（单传感器MAE）
  - 全监督基线
- **核心结果**：
  - MOMO整体性能优于所有对比基线
  - 分割任务上表现"一致且显著的提升"（consistent and significant performance improvement）
  - 分类任务上达到可比较的性能（comparable performance）

### Q8: 代码开源？
<回答>

🎯 结论：代码已完全开源，提供模型权重、预训练代码、预训练数据和评估代码。

📌 要点：
- **开源地址**：https://github.com/kerner-lab/MOMO
- **开源内容**：
  - 模型权重（MOMO foundation model weights）
  - 预训练代码（pretraining code）
  - 预训练数据（pretraining data）
  - 评估代码（evaluation code）
- **论文表述**："The model weights, pretraining code, pretraining data, and evaluation code are available at: this https URL"（摘要末尾）以及"available at: github.com/kerner-lab/MOMO"（摘要最后一句）

### Q9: 客观评价？
<回答>

🎯 结论：MOMO是火星遥感基础模型领域的开创性工作，方法创新性强且实验充分，但在传感器扩展性和任务覆盖度上仍有提升空间。

📌 要点：
- **优势一：问题定义清晰，针对性强**：准确识别火星遥感数据"极端分辨率差异"和"覆盖范围不均"的核心挑战，提出的独立预训练+模型合并策略直击痛点
- **优势二：方法创新度高**：EVL检查点选择策略和感知-结构联合损失均为首次提出，具有方法论价值；模型合并范式为多传感器基础模型构建提供新思路
- **优势三：实验系统全面**：覆盖3个传感器、9个下游任务、多类强基线对比，且完全开源，可复现性强
- **短板一：传感器覆盖有限**：仅整合3个传感器，未包含火星其他重要传感器（如CRISM高光谱、MOLA地形等），多模态扩展性待验证
- **短板二：下游任务类型局限**：Mars-Bench以分类和分割为主，缺乏目标检测、变化检测、三维重建等更复杂的火星应用任务验证

### Q10: 批判审视？
<回答>

🎯 结论：MOMO的技术来源多元且落地价值明确，但在理论深度和实际部署上仍有优化空间。

📌 要点：
- **技术来源分析**：
  - 基础架构：Masked Autoencoder（MAE）来自计算机视觉领域
  - 模型合并：任务算术（task arithmetic）来自模型编辑研究
  - 检查点选择：验证损失对齐策略受模型训练动态分析启发
  - 损失设计：LPIPS、SSIM、梯度损失均为图像重建领域成熟技术
  - 整体为"成熟技术+新颖组合+领域适配"的创新模式
- **落地价值**：
  - 科学价值：为行星科学家提供低成本定制模型训练的基础能力，加速火星地质分析
  - 工程价值：模型合并策略支持新传感器灵活接入，避免重复预训练，适合长期任务演进
  - 应用价值：在撞击坑制图、滑坡监测、巨石定位等任务中已验证有效性
- **潜在改进方向**：
  - **理论深化**：EVL策略目前基于验证损失数值匹配，可探索损失曲率、Hessian矩阵等更精细的收敛状态度量
  - **效率优化**：三传感器独立预训练+合并流程计算开销较大，可研究渐进式合并或在线学习策略
  - **模态扩展**：整合CRISM高光谱、MOLA高程数据，构建真正的多模态火星基础模型
  - **下游增强**：在更多复杂任务（时序变化检测、三维表面重建）上验证并优化特征提取能力

---

Powered by OpenClaw🦞
